app/api/dao/task.py

```diff
@@ -49,24 +56,29 @@ def create_task(user_id, mentorship_relation_id, data):
     def list_tasks(user_id, mentorship_relation_id):
         """Retrieves all tasks of a user in a mentorship relation.
 
-        Lists all tasks from a mentorship relation for the specified user if the user is involved in a current mentorship relation.
+        Lists all tasks from a mentorship relation for the specified user
+        if the user is involved in a current mentorship relation.
 
         Args:
             user_id: The id of the user.
             mentorship_relation_id: The id of the mentorship relation.
 
         Returns:
-            A list containing all the tasks one user has in a mentorship relation. otherwise, it returns a two element list where the first element is
-            a dictionary containing a key 'message' indicating in its value if there were any problem finding user's tasks in the specified
-            mentorship relation as a string. The last element is the HTTP response code
+            A list containing all the tasks one user has in a
+            mentorship relation. Otherwise, it returns a two
+            element list where the first element is
+            a dictionary containing a key 'message' indicating in its
+            value if there were any problem finding user's tasks in the
+            specified mentorship relation as a string.
+            The last element is the HTTP response code
         """
 
         user = UserModel.find_by_id(user_id)
         relation = MentorshipRelationModel.find_by_id(mentorship_relation_id)
         if relation is None:
             return messages.MENTORSHIP_RELATION_DOES_NOT_EXIST, 404
 
-        if not (user_id == relation.mentee_id or user_id == relation.mentor_id):
+        if user_id != relation.mentee_id and user_id != relation.mentor_id:
```

Comment:
Hey @devmrfitz , can you explain these changes?

Reply:
I converted (not(A or B)) into ((not A) and (notB)) 
This shortened the length of the line as directed by pycodestyle/pep8